python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -950,21 +950,23 @@ def _apply_to_members(self, fn):
 
         return self
 
-    def to(self, device: torch.device) -> None:  # pylint: disable=invalid-name
+    def to(
```

Comment:
@frozenbugs if we can merge these changes in a separate PR, the examples will be runnable with DGL nightly.

Reply:
Yes, let's merge this in a separate PR, the change LGTM.